invenio_app_ils/ui/src/invenio_app_ils/common/components/SearchControls/components/SearchSortOrder/SearchSortOrder.js

```diff
@@ -35,8 +35,8 @@ export default class SearchSortOrder extends Component {
           <Icon
             name={
               element.value === 'asc'
-                ? 'sort numeric ascending'
-                : 'sort numeric descending'
+                ? 'sort alphabet ascending'
```

Comment:
I think the word should be `alphabetically`

Reply:
that's how semantic UI names their icons :)